topic-offset-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/topicoffset/kafka/KafkaContractTest.java

```diff
@@ -85,8 +85,8 @@
 
 /**
  * An external requirement of the service is to connect to a pair of kafka topics.
- * The upstream "attribute-masking-service" topic is and read by this service.
- * The downstream "filtered-resource-offset" topic is written to by this service.
+ * The upstream "masked-resource" topic is written to by the attribute-masking-service and read by this service.
+ * The downstream "masked-resource-offset" topic is written to by this service and read by the filtered-resource-service.
```

Comment:
```suggestion
 * The downstream "masked-resource-offset" topic is written to by this service and read by the Filtered-Resource-Service.
```

Reply:
Capitalising the words would deviate with the convention.